language/move-prover/interpreter-testsuite/tests/concrete_check/choice.move

```diff
@@ -1,6 +1,20 @@
 module 0x2::A {
+    use std::signer;
     use std::vector;
 
+    struct R has key { i: u64 }
+
+    #[test(a=@0x2)]
+    public fun move_to_test(a: &signer) {
```

Comment:
For consistency, maybe call this test 'unroll_address_satisfiable'? And correspondingly, there should be an unsatisfiable case?

Reply:
Make a lot of sense! Renamed and added test case